ksqldb-execution/src/main/java/io/confluent/ksql/execution/util/ExpressionTypeManager.java

```diff
@@ -233,10 +234,15 @@ public Void visitComparisonExpression(
       final SqlType rightSchema = context.getSqlType();
 
       if (!ComparisonUtil.isValidComparison(leftSchema, node.getType(), rightSchema)) {
-        throw new KsqlException("Cannot compare "
-            + node.getLeft().toString() + " (" + leftSchema.toString() + ") to "
-            + node.getRight().toString() + " (" + rightSchema.toString() + ") "
-            + "with " + node.getType() + ".");
+        throw new KsqlStatementException(
+            "Cannot compare " + leftSchema + " to " + rightSchema + " "
+            + "with " + node.getType() + ".",
```

Comment:
What is an example of `node.getType()` here?

Reply:
It's something like `GREATER_THAN_OR_EQUAL`.